server/db/index.js

```diff
@@ -1,27 +1,56 @@
-const mongoose = require('mongoose');
-mongoose.connect('mongodb://localhost/fandangit', {useNewUrlParser: true})
-
-const movieSchema = mongoose.Schema({
-  id: Number,
-  name: String,
-  associatedVideos: Array
-})
-
-let Movie = mongoose.model('Movie', movieSchema)
-
-let getMovieData = (id, callback) => {
-  let query = Movie.find({id: id});
-
-  query.exec((err, docs) => {
-    if (err) {
-      console.log('error querying mongodb: ', err);
-      callback(err, null);
-    } else {
-      console.log('mongodb query successful!');
-      callback(null, docs);
-    }
+// const mongoose = require('mongoose');
+// mongoose.connect('mongodb://localhost/fandangit', {useNewUrlParser: true})
+
+// const movieSchema = mongoose.Schema({
+//   id: Number,
+//   name: String,
+//   associatedVideos: Array
+// })
+
+// let Movie = mongoose.model('Movie', movieSchema)
+
+// let getMovieData = (id, callback) => {
+//   let query = Movie.find({id: id});
+
+//   query.exec((err, docs) => {
+//     if (err) {
+//       console.log('error querying mongodb: ', err);
+//       callback(err, null);
+//     } else {
+//       console.log('mongodb query successful!');
+//       callback(null, docs);
+//     }
+//   })
+// }
+
+// module.exports.getMovieData = getMovieData;
+
+const { Pool } = require('pg');
+const pool = new Pool({
+  user: 'postgres',
+  host: '127.0.0.1',
+  database: 'mydb',
+  password: '',
+  port: 5432,
+});
+
```

Comment:
Like Robin mentioned in my repo, factoring these configurations out may be useful